lms/static/js/views/fields.js

```diff
@@ -383,7 +388,16 @@
 
             updateValueInField: function() {
                 var value = (_.isUndefined(this.modelValue()) || _.isNull(this.modelValue())) ? '' : this.modelValue();
-                this.$('.u-field-value input').val(value);
+
+                var fieldHasFocus = (document.activeElement === this.$('.u-field-value input')[0]);
+                var fieldChanged = this.fieldValue() !== value;
+                if (fieldHasFocus && fieldChanged) {
```

Comment:
Is it possible for us to just tweak the conditionals here to avoid the empty `if` statement?  Same on the other one below.  
Also, a tiny nit, on the next line "condition" is misspelled.

Reply:
Sure, @davidjoy , I'll be handling this 👍